tests/api/v1/generators/get.js

```diff
@@ -191,4 +192,32 @@ describe(`api: GET ${path}`, () => {
       done();
     });
   });
+
+  it('find by tags, found', (done) => {
+    api.get(`${path}?tags=status`)
+    .set('Authorization', token)
+    .expect(constants.httpStatus.OK)
+    .end((err, res) => {
+      if (err) {
+        done(err);
+      }
+
+      expect(res.body).to.have.length(FOUR);
+      done();
+    });
+  });
+
+  it('find by tags, not found', (done) => {
+    api.get(`${path}?tags=blah`)
+    .set('Authorization', token)
+    .expect(constants.httpStatus.OK)
+    .end((err, res) => {
+      if (err) {
+        done(err);
```

Comment:
We can take care of this later but we want to start adding a return in front of done(err). This will avoid the "done() called multiple times" error message when the test fails.